LethalAPI.Core/Events/Features/Patcher.cs

```diff
@@ -164,11 +164,50 @@ public void UnpatchAll()
     /// Gets all types that have a <see cref="HarmonyPatch"/> attributed to them.
     /// </summary>
     /// <returns>A <see cref="HashSet{T}"/> of all patch types.</returns>
-    private static HashSet<Type> GetAllPatchTypes() => Assembly.GetExecutingAssembly().GetTypes().Where((type) => type.CustomAttributes.Any((customAtt) => customAtt.AttributeType == typeof(HarmonyPatch))).ToHashSet();
+    private static HashSet<Type> GetAllPatchTypes()
+    {
+        HashSet<Type> types = new HashSet<Type>();
```

Comment:
Can you change this to either
```cs
HashSet<Type> types = new();
```
or? 
```cs
var types = new HashSet<Type>();
```

Reply:
This came from the original code, but I feel like the method is fair being that it enumerates through a fairly hefty amount of types. I will change it to brace style for clarity.

Reply:
Also I think we shouldn't use car and instead we should always use explicitly defined types.

Reply:
After looking through the code I now remember it also has to be a method as it must catch the types that have dependencies ie the bepinex patches.

Reply:
As for the new, I will update accordingly.